frameworks/file-analysis.rst

```diff
@@ -23,6 +23,19 @@ File Analysis
     provide analysis specifically for files that is analogous to the
     analysis Zeek provides for network connections.
 
+Supported Protocols
```

Comment:
Should this mention that these are the protocols supported by default but not necessarily all of them (i.e. packages)?

Reply:
Yeah, agree. Will do. I'm on #91 right now where I've state the same thing for the file analyzes themselves (like entropy, PE, etc).